.github/CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ Please send a pull request to the master branch. Please include [documentation](
 - Separate code commits from reformatting commits.
 - Provide tests for any newly added code.
 - Follow PEP8.
-- When committing only documentation changes please include [ci skip] in the commit message to avoid running tests on Travis-CI.
+- When committing only documentation changes please include [ci skip] in the commit message to avoid running tests on Travis-CI and AppVeyor.
```

Comment:
Anything in the `script:` section of [.travis.yml](https://github.com/python-pillow/Pillow/blob/master/.travis.yml#L46-L56) that returns with an error code will fail the build. The docs build is in that section, but I don't know if it returns error codes for problems.